package.json

```diff
@@ -52,6 +52,10 @@
     "generate:example": "node bin/example-generator/index.js",
     "generate:docs": "rimraf docs/ && jsduck --config jsduck.json",
     "generate:docs-w-ext": "rimraf docs-w-ext/ && jsduck --config jsduck-w-ext.json",
+    "generate:docs:master": "jsduck --config=jsduck.json --output=master/docs --title=\"GeoExt $npm_package_version-dev Documentation\" --warnings=-inheritdoc",
```

Comment:
This basically does the same as the existing `generate:docs` / `generate:docs-w-ext`, I suggest that the 'older' ones can be removed.

Reply:
During CI we'll need `gem install jsduck` or some ruby action :man_shrugging:  not an expert.

Reply:
Where does `$npm_package_version` come from?

Reply:
` --warnings=-inheritdoc` shoudl be moved to configfile `jsduck.json`

Reply:
Right, forgot that this was a ruby gem...

* the `$npm_package_version` is something I recently learned ;-) Those are automatically set by npm, you can access the whole `package.json` if I understand this correctly
* the `--warnings=-inheritdoc` cannot be moved to the config file as it is different for the different invocations

Reply:
Cool, thx

Reply:
Fixed in 6a24509b